wandb/data_types.py

```diff
@@ -290,25 +344,46 @@ def add_data(self, *data):
                     len(self.columns), self.columns
                 )
             )
-        self._validate_data(data)
+        data = list(data)
```

Comment:
Do you need to convert to a list here? enumerate should work on any iterable.

Reply:
since the *data parameter wraps the parameters in a tuple, i think it is needed :/

Reply:
fixed